sources/platform/runtime/machine-image/language-versions.md

```diff
@@ -165,6 +165,8 @@ The following images are available when you configure `language: go` in your [sh
 	|1.2         |  - Ubuntu 14.04<br>- Ubuntu 16.04 | [v5.6.1](/platform/runtime/machine-image/ami-v561/) and earlier |
 	|1.1         |  - Ubuntu 14.04<br>- Ubuntu 16.04 | [v5.6.1](/platform/runtime/machine-image/ami-v561/) and earlier |
 
+	Note : Golang builds on Shippable using `gvm` which will take care of installing any version of golang provided to it, or it will use the one already installed on the machine.
```

Comment:
We need to tweak the language for clarity:

Shippable uses `gvm` to manage multiple versions of Go in this image. If your build is configured to use a version of Go that is not already present in the image, `gvm` will attempt to install it at runtime.

Reply:
Please update the other notes to follow this pattern.